src/cljs/athens/views/app_toolbar.cljs

```diff
@@ -108,7 +212,15 @@
        (when @merge-open?
          [filesystem/merge-modal merge-open?])
 
-       [:header (use-style app-header-style)
+       [:header (use-style (cond
+                             (and (= (util/get-os) :mac) electron?) mac-app-header-style
+                             (and (= (util/get-os) :windows) electron?) win-app-header-style
+                             (and (= (util/get-os) :linux) electron?) linux-app-header-style)
```

Comment:
Use `case` instead of `cond`. `case` also throws an error if no case is matched, unlike `cond` which just returns `nil.